src/System.IO.MemoryMappedFiles/tests/MemoryMappedViewStream/ReadWrite.cs

```diff
@@ -339,143 +339,140 @@ public bool runTest()
                 }
             }
 
-            if (Interop.PlatformDetection.OperatingSystem == Interop.OperatingSystem.Windows) // currently don't support sharing views of pagefile backed MMFs on Unix
+            // Use a pagefile backed MMF instead of file backed
+            using (MemoryMappedFile mmf = MemoryMappedFile.CreateNew(null, pageSize * 10))
```

Comment:
GitHub isn't doing a great job with this diff, but the only thing I've changed in this file is removing this first line that special-cased the test to be for Windows; the rest of the shown changes are because the indentation then changed.